### ISSUE TITLE
Better to return 2 parameters, to know the function success or error

### ISSUE THREAD

**Issue by Arif9878** [smilelikeshit/go-build-image#2]
https://github.com/smilelikeshit/go-build-image/blob/9fb2fc4e0b876e13154e9dffa93881a6ac83aa74/main.go#L146

**Comment by smilelikeshit** [comment]
what a function shoud i put to get success or error ? . u mean conditional to check param image, name is available or not ?

**Comment by Arif9878** [comment]
yes it's better if you make a function there return success or error, to make things easier tracing error.
